autox/utils/utils.py

```diff
@@ -0,0 +1,187 @@
+
+import torch
+import torch.nn as nn
+
+import math
+import numpy as np
+import pandas as pd
+
+from natsort import natsorted
+from collections import OrderedDict
+
+class AverageMeter:
```

Comment:
Will remove __main__, it was used for development purpose. Regarding the ```extra_repr```, can we push it ahead of this pull? as far as i see, printing of quantizer attributes (mode, channel, bitwidth) is used as we move towards mixed precision now.

Reply:
If you need printing of quantizer attributes, it's better to have an explicit method for this in BaseQuantizer, like `__str__`  instead of an implicit setting it:
```
setattr(BaseQuantizer, 'extra_repr', extra_repr)
```

Reply:
High priority. Please discuss

Reply:
quantizer_tracing.py has been removed from project. extra_repr has been moved to BaseQuantizer class definition.